external-crates/move/solana/move-mv-llvm-compiler/src/cli.rs

```diff
@@ -23,6 +23,11 @@ pub struct Args {
     #[clap(long = "skip-basic-blocks")]
     pub skip_basic_blocks: bool,
 
+    /// Skip undefined entry function.
+    // This allows to compile Sui pacages with generic entry point functions
+    #[clap(long = "skip-undefined-entries")]
+    pub skip_undefined_entries: bool,
```

Comment:
If set true by default it will compile with a warning, but a user will likely ignore this warning and only later discover incompleteness.
I think it is better to fail in the first compilation and then let user to decide if it is ok to ignore some entry points.

Also I do not expect we will need generic entry points in Solana, so this option is only for our testing of Sui framework.